clients/src/main/java/org/apache/kafka/common/record/FileLogInputStream.java

```diff
@@ -309,7 +241,9 @@ public int hashCode() {
 
         @Override
         public String toString() {
-            return "FileChannelRecordBatch(magic: " + magic() + ", offsets: [" + baseOffset() + ", " + lastOffset() + "])";
+            return "FileChannelRecordBatch(magic: " + magic +
+                    ", offset: " + offset +
+                    ", size: " + batchSize + ")";
```

Comment:
Hmm.. On second though, I was trying to avoid any incidental loading from `toString`.